env.d.ts

```diff
@@ -16,5 +16,6 @@ declare namespace NodeJS {
     BUGSNAG_API_KEY: string;
     NEXT_PUBLIC_DEBUG: string;
     NEXT_PUBLIC_APP_VERSION: string;
+    NEXT_PUBLIC_SHOW_TEST_VIEW: string;
```

Comment:
less action more declaration: NEXT_PUBLIC_TEST_VIEW_VISIBLE

Reply:
this is deleted in the next pull request